scripts/vagrant/provision/setup_privileged.sh

```diff
@@ -3,15 +3,33 @@
 # Use with Ubuntu 16.x+
 set -xe
 
+apt-get update
+
+DOCKER_CONFIG_FILE=daemon.json
```

Comment:
Does this file exist already? Ah I see it https://github.com/m3db/m3/blob/e0a408895c6d98b20a42c8687295dc194ffbe0ea/scripts/vagrant/docker/daemon.json